components/browser/menu/src/test/java/mozilla/components/browser/menu/item/WebExtensionBrowserMenuItemTest.kt

```diff
@@ -161,6 +162,40 @@ class WebExtensionBrowserMenuItemTest {
         assertEquals(View.INVISIBLE, badgeView.visibility)
     }
 
+    @Test
+    @Ignore("WIP")
+    fun `every time we have a badgeBackgroundColor we are setting tint`() {
+        val icon: Bitmap = mock()
+        val imageView: ImageView = mock()
+        val badgeView = TextView(testContext)
+        val labelView: TextView = mock()
+        val container = View(testContext)
+        val view: View = mock()
+
+        whenever(view.findViewById<ImageView>(R.id.action_image)).thenReturn(imageView)
+        whenever(view.findViewById<TextView>(R.id.badge_text)).thenReturn(badgeView)
+        whenever(view.findViewById<TextView>(R.id.action_label)).thenReturn(labelView)
+        whenever(view.findViewById<View>(R.id.container)).thenReturn(container)
+        whenever(view.context).thenReturn(testContext)
+
+        val browserAction = Action(
+                title = "title",
+                loadIcon = { icon },
+                enabled = true,
+                badgeText = "badgeText",
+                badgeTextColor = Color.WHITE,
+                badgeBackgroundColor = Color.BLUE
+        ) {}
+
+        val action = WebExtensionBrowserMenuItem(browserAction) {}
+        action.bind(mock(), view)
+        testDispatcher.advanceUntilIdle()
+
+        badgeView.background?.setTint(Color.BLUE)
+        val badgeViewBackground = badgeView.background as ColorDrawable
```

Comment:
Oh, here the background is not getting initialized only casted.

Reply:
Do not worry I noticed that the bind test was already testing this functionality

Reply:
Take a look at to the snipped above the same solution could be applied here.

Reply:
Ohhh, we're gonna remove this test? Ok, how we're gonna improve bind test?

Reply:
The test looks good we can keep it, it would be an additional check. But we definitely need to improve the test `bind` as now it not verifying see `verify(badgeView).background?.setTint(Color.BLUE)` the `?`  is hiding the check